independent-projects/arc/runtime/src/main/java/io/quarkus/arc/InvocationContexts.java

```diff
@@ -0,0 +1,72 @@
+package io.quarkus.arc;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Method;
+import java.util.List;
+import java.util.Set;
+import java.util.function.Function;
+import java.util.function.Supplier;
+import javax.interceptor.InvocationContext;
+
+public final class InvocationContexts {
+
+    public static final String KEY_INTERCEPTOR_BINDINGS = "io.quarkus.arc.interceptorBindings";
```

Comment:
Shouldn't we place this on some more "API/SPI looking" class? The methods on this class are otherwise purely internal whereas this key can be used by integrators (the same way that they can use the util class).
How about we put that inside `extensions/arc/runtime/src/main/java/io/quarkus/arc/runtime/InterceptorBindings.java`?

Reply:
So the problem with the `io.quarkus.arc.runtime` package is that it should only contain runtime internals. The API stuff should be located in `io.quarkus.arc` but we already use this package in the ArC project itself. And that brings me to the fact that this key is not specific for Quarkus. In theory, we can introduce a new interface similar to `WeldInvocationContext`, i.e. `ArcInvocationContext` and put the key there.

Reply:
It's probably not worth it for a single key but I get your point.